### DISCUSSION TITLE
Images cannot be displayed after upload

### DISCUSSION THREAD

**Discussion by tmighty7** [outline/outline#4171] [Self Hosting]
When I add an image to the document, it is uploaded correctly to the S3 Storage but not saved / displayed in the document. Uploading  / Saving  / Displaying Profile-Images works fine.
In the debug console it looks like there is a wrongly redirect to Amazon when uploading an image to a document.

S3 Env-Variables :
  AWS_REGION: "us-east-1"
  AWS_S3_FORCE_PATH_STYLE: "false"
  AWS_S3_UPLOAD_BUCKET_NAME: "outline"
  AWS_S3_UPLOAD_BUCKET_URL: "https://outline.s3.fz-juelich.de"
  AWS_S3_UPLOAD_MAX_SIZE: "26214400"
  AWS_ACCESS_KEY_ID: "xxxxxxxx"
  AWS_SECRET_ACCESS_KEY: "xxxxxxxx"

**To Reproduce**
1. Add Image to Document
2. after reloading the Document, the image disappeared

**Expected behavior**
The Images should be visible and saved to document

**Screenshots**
![ picture](https://user-images.githubusercontent.com/25579277/192282508-fbe15692-eba7-43e8-aab5-284864054935.png)

**Debug-Output**
![console](https://user-images.githubusercontent.com/25579277/192282682-3b07b168-1d24-4a65-8b6d-17db20e1958c.png)
![network](https://user-images.githubusercontent.com/25579277/192282711-49ad7844-286d-46dd-b914-847584193727.png)

**Outline (please complete the following information):**
- Install: self hosted, kubernetes, self-hosted S3
- Version: 0.66.1

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser Firefox
 - Version 105.0

**Comment by ChetanGoti** [comment]
I had faced this exact issue few months back when I started using image uploads. 

I was running  outline and minio behind nginx proxy and it was getting wrong `Host` header, hence the signature it calculated didn't match up with minio. As a result, requests would fail with 403.

I was able to fix it by setting correct Host headers. e.g:

```
proxy_set_header Host $host;
```
